.github/workflows/deploy_to_npm.yaml

```diff
@@ -35,7 +35,6 @@ jobs:
                   yarn test-cov
                   yarn karma-headless
                   yarn karma-headless-firefox
-                  ./scripts/test-npm-packages.sh
```

Comment:
Now we'll never test the npm packages right? Shouldn't we instead skip this step for release branch but keep doing it on master?